vendor_blueprints/export_card_to_file.py

```diff
@@ -0,0 +1,166 @@
+    card_headers = {
+            'Content-Type' : 'application/json',
+            'x-domo-authentication': access_token
+        }
+    card_response = requests.get(url=card_info_api, headers=card_headers)
+    return card_response.json()
+
+
+def export_graph_to_file(card_id, file_name, file_type, 
+                         access_token, DOMO_INSTANCE, folder_path=""):
+    """
+    Exports a file to one of the given file types: csv, ppt, excel
+    """
+    export_api = f"https://{DOMO_INSTANCE}.domo.com/api/content/v1/cards/{card_id}/export"
+
+    card_headers = {
+            'Content-Type': 'application/x-www-form-urlencoded',
+            'accept': 'application/json, text/plain, */*',
+            'x-domo-authentication': access_token
+    }
+    
+    # make a dictionary to map user file_type with requested mimetype
+    filetype_map = {
+        "csv":"text/csv", 
+        "excel":"application/vnd.openxmlformats-officedocument.spreadsheetml.sheet",
+        "ppt": "application/vnd.ms-powerpoint"
+    }
+    
+    body = {
+        "queryOverrides":{
+            "filters":[],
+            "dataControlContext":{
+                "filterGroupIds":[]
+            }
+        },
+        "watermark":'true',
+        "mobile":'false',
+        "showAnnotations":'true',
+        "type":"file",
+        "fileName": f"{file_name}",
+        "accept": filetype_map[file_type]
+    }
+    # convert body to domo encoded payload
+    encoded_body = urllib.parse.quote(f"{body}")
+    encoded_body = encoded_body.replace("%27", "%22") # changing " to '
+    payload = f"request={encoded_body}"
+
+    export_response = requests.post(url=export_api, data=payload,
+        headers=card_headers, stream=True)
+    if export_response.status_code == 200:
+        destination_folder_name = shipyard.files.clean_folder_name(
+            folder_path)
+        destination_full_path = shipyard.files.combine_folder_and_file_name(
+        folder_name=destination_folder_name, file_name=file_name)
+        with open(destination_full_path, 'wb') as fd:
+            # iterate through the blob 1MB at a time
+            for chunk in export_response.iter_content(1024*1024):
+                fd.write(chunk)
+        print(f"{file_type} file:{destination_full_path} saved successfully!")
+    else:
+        print(f"Request failed with status code {export_response.status_code}")
+        sys.exit(EXIT_CODE_BAD_REQUEST)
+
+
+def main():
+    args = get_args()
+    email = args.email
+    password = args.password
+    card_id = args.card_id
+    file_name = args.dest_file_name
+    folder_path = args.dest_folder_path
+    file_type = args.file_type
+    folder_path = args.folder_path
+    DOMO_INSTANCE = args.domo_instance
+    ACCESS_TOKEN = get_access_token(email, password, DOMO_INSTANCE)
+    # check if the card is of the 'dataset/graph' type
+    card = get_card_data(card_id, ACCESS_TOKEN, DOMO_INSTANCE)[0]
```

Comment:
It still feels weird that you've named these as if they are constants. Can you rename to lower case and treat like the rest of the argument variables. This includes where they are used in the other functions.